test/utils/clean/clean.go

```diff
@@ -57,9 +58,11 @@ func Configs() {
 
 	for _, ptpConfig := range ptpconfigList.Items {
 		if ptpConfig.Name == utils.PtpGrandMasterPolicyName ||
```

Comment:
This check ensure only test configured PTPconfigs are cleaned , right ?

Reply:
yes, correct